test/integration/features/basic.feature

```diff
@@ -29,6 +29,10 @@ Feature: Basic
     Given Minishift has state "Does Not Exist"
      When executing "minishift start" succeeds
      Then Minishift should have state "Running"
+      And image export completes with 3 images
+      And container image "openshift/origin-haproxy-router:v3.6.1" is cached
+      And container image "openshift/origin-docker-registry:v3.6.1" is cached
+      And container image "openshift/origin:v3.6.1" is cached
```

Comment:
These all should have a regex not hardcoded tag like v3.6.1 because right now it works as our default OpenShift version is 3.6.1 but when we update our default version then it will fail 👎

Reply:
+1 for regex